webExtensionApis/testingOverrides/stubApi.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+
+ChromeUtils.import("resource://gre/modules/ExtensionCommon.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionUtils.jsm");
+
+// eslint-disable-next-line no-undef
+const { EventManager } = ExtensionCommon;
+// eslint-disable-next-line no-undef
+const { EventEmitter } = ExtensionUtils;
+
+this.testingOverrides = class extends ExtensionAPI {
```

Comment:
My suggestion is to have this be json rather than 20 different prefs.   People might have opinions on this though.  

It's also a smaller api :  "getTestingOverrides(). => {firstRunTs, blah blah}"

Reply:
Moved discussion to https://github.com/mozilla/shield-studies-addon-utils/issues/212